CMakeLists.txt

```diff
@@ -158,6 +158,21 @@ if (build_timing)
   #set_property(GLOBAL PROPERTY RULE_LAUNCH_CUSTOM "${CMAKE_COMMAND} -E time")
 endif()
 
+#--- Set up address sanitizer builds ----------------------------------------------------------
+if(asan)
+  if(NOT CMAKE_COMPILER_IS_GNUCXX AND NOT CMAKE_CXX_COMPILER_ID MATCHES Clang)
+    message(WARNING "Address sanitizer builds only work with GCC and Clang")
```

Comment:
is it ok to keep going anyway?

Reply:
Rephrased to "only tested". Maybe it will work, but we don't promise anything.